test/analysis/expected/expected_df.csv

```diff
@@ -0,0 +1,8 @@
+,test,test,test
```

Comment:
We should add a column header for the edge column

Reply:
This happens to be the index